ui/app/services/api.ts

```diff
@@ -83,6 +84,39 @@ export default class ApiService extends Service {
     return { url, init };
   };
 
+  normalizeRequestBodyKeys = async (context: RequestContext) => {
```

Comment:
The client typically converts the keys to snake from camel based on the schema defined in the OpenAPI spec. The problem is that nested objects do not have a schema (config for example) so the keys are not transformed. I added this method to ensure that all of the request body keys are always snake case.

Reply:
No all the keys sent in the request body (POST, PUT, PATCH) will be snake cased. I had added a similar method to the api client a while back to do the opposite for incoming requests and convert the keys to camelCase.